spec/lib/audit/catalog_to_moab_instance_spec.rb

```diff
@@ -133,6 +223,56 @@
         expect(pohandler).to receive(:update_version_after_validation)
         c2m.check_catalog_version
       end
+
+      context 'check whether PreservedCopy already has a status other than OK_STATUS, re-check status if so' do
+        it 'had OK_STATUS, version increased, should still have OK_STATUS' do
```

Comment:
this is another place to consider condensing code to read without reducing the tests executed?